sos/plugins/services.py

```diff
@@ -9,26 +9,23 @@
 from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
 
 
-class Services(Plugin):
+class Services(Plugin, DebianPlugin, UbuntuPlugin):
     """System services
     """
 
     plugin_name = "services"
     profiles = ('system', 'boot')
 
-    option_list = [("servicestatus", "get a status of all running services",
-                    "slow", False)]
-
     def setup(self):
         self.add_copy_spec([
             "/etc/inittab",
-            "/etc/rc.d"
+            "/etc/rc.d",
+            "/etc/rc*.d"
```

Comment:
What's the structure on Debian? On Red Hat SysV we have `/etc/rc.d/`, and all the `/etc/rcX.d` are simply symlinks into the correspondingly named directory in `rc.d`.

```
# ll /etc/rc*.d -d
lrwxrwxrwx.  1 root root   10 May  6  2016 /etc/rc0.d -> rc.d/rc0.d
lrwxrwxrwx.  1 root root   10 May  6  2016 /etc/rc1.d -> rc.d/rc1.d
lrwxrwxrwx.  1 root root   10 May  6  2016 /etc/rc2.d -> rc.d/rc2.d
lrwxrwxrwx.  1 root root   10 May  6  2016 /etc/rc3.d -> rc.d/rc3.d
lrwxrwxrwx.  1 root root   10 May  6  2016 /etc/rc4.d -> rc.d/rc4.d
lrwxrwxrwx.  1 root root   10 May  6  2016 /etc/rc5.d -> rc.d/rc5.d
lrwxrwxrwx.  1 root root   10 May  6  2016 /etc/rc6.d -> rc.d/rc6.d
drwxr-xr-x. 10 root root 4096 Apr 28  2017 /etc/rc.d
```

Reply:
Also symlinks, but not the directories;
$ ll /etc/rc*.d
/etc/rc0.d:
total 16
drwxr-xr-x   2 root root  4096 Nov 23 13:50 ./
drwxr-xr-x 149 root root 12288 Nov 22 09:55 ../
lrwxrwxrwx   1 root root    20 Apr 14  2018 K01alsa-utils -> ../init.d/alsa-utils*
lrwxrwxrwx   1 root root    13 Apr 19  2018 K01atd -> ../init.d/atd*
lrwxrwxrwx   1 root root    22 Apr 14  2018 K01avahi-daemon -> ../init.d/avahi-daemon*

I figure there's no harm in making sure whatever symliks should be there are there on all distros?